MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -88,13 +88,13 @@ class OpenTKGamePlatform : GamePlatform
 		private OpenALSoundController soundControllerInstance = null;
         // stored the current screen state, so we can check if it has changed.
         private bool isCurrentlyFullScreen = false;
-        private Toolkit toolkit;
         private int isExiting; // int, so we can use Interlocked.Increment
+
+        int windowDelay = 11;
```

Comment:
Just seems fragile to depend on message pumps that are often inconsistent in timing and order to wait some exact number of updates.

Also why isn't OpenTK handling this for us?  It shouldn't be our job to guess when it is safe to dispose their window.